model.py

```diff
@@ -1,150 +1,261 @@
+import sqlite3
+import json
+import random
+
 class Model:
-	def __init__(self):
-		'''
-		Constructor 
-		'''
-		pass
-
-	def set_user_info(self, nutrition_standard,
-                           height,
-                           weight,
-                           age,
-                           meal_list):
-        """
-        Store user info from surveys to database
-        @param
-        nutrition_standard: String
-        height: Int
-        weight: Int
-        age: Int
-        meal_list: List<String>
-
-        Database: UserInfo
-        @column
-        nutrition_standard: String
-        height: Int
-        weight: Int
-        age: Int
-        meal_list: List<String>
-        """
+    connection  = sqlite3.connect('mealplan.db')
+    cursor = connection.cursor() 
+    
+    #the meal_list has to be string because sqlite cannot store list datatype
+    #then using the json library to deserialize the meal list into string type
+
+    command_createUserInfo = '''
+        CREATE TABLE IF NOT EXISTS 
+        UserInfo(ID INTEGER PRIMARY KEY AUTOINCREMENT, nutrition_standard TEXT, height INTEGER, weight INTEGER, age INTEGER, meal_list TEXT)
+    ''' 
+
+    command_createUserDefinedMeal = ''' 
+        CREATE TABLE IF NOT EXISTS
+        UserDefinedMeal(meal_name TEXT, start_time TEXT, end_time TEXT, 
+                        set_of_dishes TEXT, nutritious_retriction TEXT, 
+                        regular BOOLEAN, flexible BOOLEAN)
+    ''' 
+
+    command_createRecipe = '''
```

Comment:
should change to things like `create_recipe_command`

Reply:
changed!